docs-3.x/guides/setup-lando-on-windows-with-wsl-2.md

```diff
@@ -70,6 +70,55 @@ You'll want to do all of your shell activity from within the WSL system, and you
 ## Some caveats
 
 Performance is generally very good with this approach, however, stability can at times leave something to be be desired. Occassional freezes under heavy load are fairly commonplace, and seem to be resolved by rebooting the system. All things considered, the reduction in CPU load, battery drain, and the accompanied increase in the speed of every single action you take (accessing site pages, running any CLI commands) more than outweighs the disruption of these occasional issues for most users. Hopefully stability will continue to increase with time.
+            
+## Configure host IP
+
+If you are running Lando inside WSL you have to override the `LANDO_HOST_IP` to use Xdebug because by default Lando uses the WSL IP which can switch between reboots of WSL.
+Also, you have to differentiate between running your IDE (e.g. PHPStorm) in Windows or inside WSL.
+
+### 1. Add this lines to `~/.bashrc` inside WSL2 :
+a) For IDE in Windows:
+```bash
+// ~/.bashrc
+# Set correct dev host to Windows
+export LANDO_HOST_NAME_DEV=host.wsl.internal
+export LANDO_HOST_GATEWAY_DEV=$(cat /etc/resolv.conf | grep nameserver | awk '{print $2; exit;}')
```

Comment:
Unfortunately I don't have a better solution - in particular you need to rebuild if wsl changes windows host ip address! Are there any better suggestions to get the windows host ip?